windows/include/gettext.h

```diff
@@ -0,0 +1,288 @@
+/* Convenience header for conditional use of GNU <libintl.h>.
+   Copyright (C) 1995-1998, 2000-2002, 2004-2006, 2009-2013 Free Software
+   Foundation, Inc.
+
+   This program is free software; you can redistribute it and/or modify
```

Comment:
What are the implications of including GPLed code? Everyone can build it but if you redistribute it then you will have to comply with the GPL. Is this only used for the tools?
Might be fine, because there are also other ways to build for windows that should not have this problem.

We probably need some README that explains this situation and warns the user